lib/builderator/config.rb

```diff
@@ -47,10 +47,13 @@ def prepend_json(path)
         layers.unshift(File.from_json(path)) if ::File.exist?(path)
       end
 
-      def compile(max_iterations = 4)
+      def compile(max_iterations = 6)
```

Comment:
Any particular motivation for increasing this? I'm not objecting, it's just odd to me to see a change in a magic number without explanation.